src/activities/letter-in-word/Card.qml

```diff
@@ -41,6 +41,17 @@ Item {
         //visible: index % 2 != 0 ? false : true
     }
 
+    Image {
+        id: tick
+        source: "qrc:/gcompris/src/core/resource/apply.svg"
+        sourceSize.width: cardImage.width / 3
+        visible : false
+
+        anchors {
+            left: cardItem.right - 0.01
```

Comment:
you should use {left,right}Margin when you want a shift.
This is what causes the warnings:
qrc:/gcompris/src/activities/letter-in-word/Card.qml:52:21: Unable to assign double to QQuickAnchorLine